### ISSUE TITLE
Restart Deploy Job once if failed

### ISSUE THREAD

**Issue by DerMuffin** [MehdiRtal/MicrosoftRewardsBot#18]
Hey,

would it be possible to add a feature that re-run the Job (Github Workflow) once if its failed?

That terminates some issues that happend when it only run once like server connection errors.

Thanks :)

**Comment by klinix5** [comment]
Last night I was looking for some porn of the not gay variety.  Obviously gay porn is gay, so instead I went looking for guy on girl porn. But I realized that features a guy, with a penis, and is therefore 50% gay.  So I started looking for some girl on girl action, till I realized that is lesbian, and is therefore 100% gay. 

So clearly, I had to look into alternative options.  I started to consider hentai: since no one in it is real, it can’t contain guys, so it’s not gay, right? Well, hentai is anime, which not only is gay, but also makes me a weeb. The worst option so far.

I considered traps. I’ve long heard debate over whether traps are gay, so I decided to do the math.  The gayness of traps lies in the premise. Let’s consider the 5 base premises:

a guy and a trap, where the guy initially thinks he’s with a girl: effectively guy on girl, 50% gay

a guy with a trap, and the guy knew it was a trap: Dude looking for some dick, that’s gay

a girl with a trap, where the girl doesn’t know it’s a trap: that’s lesbian, and is gay

a girl with a trap, where the girl knows it’s a trap: effectively guy on girl, therefore 50% gay Therefore, on average traps are 75% gay, with a minimum gayness of 50%. Not an improvement.  The 5th option, trap on trap, is a singularity of the 4 prior possibilities, and is therefore on average 75% gay again.

So I got into specifics. What if it was girl on girl, and the premise was at least one of the girls was just bi-curious, and not actually fully lesbian? Well, in that case, their curiosity makes them a cat. Cats are furries, and furries are gay

So how about a girl soloing? Well that’s a girl touching girl bits. Lesbian and gay.

My conclusion is that the only way to live a life free of homosexual sin is to liberate oneself of sexual desire at all, and become an asexual.

Thanks for coming to my Ted Talk.
